fuse/ifuse/Portfile

```
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            libimobiledevice ifuse 1.1.4
+github.tarball_from     releases
```

Let's add a `revision` field:

```suggestion
github.tarball_from     releases
revision                0
```